api/openapi/api.yaml

```diff
@@ -179,3 +179,146 @@ paths:
           description: Check schematic log
       tags:
         - Model Operations
+  /storage/projects:
+    get:
+      summary: Get all storage projects the current user has access to
+      description: Gets all storage projects the current user has access to, within the scope of the 'storageFileview' attribute.
+      operationId: api.routes.get_storage_projects
+      parameters:
+        - in: query
+          name: input_token
+          schema:
+            type: string
+            nullable: false
+          description: Token
+          example: Token
+          required: true
+        - in: query
+          name: syn_master_file_view
+          schema:
+            type: string
+            nullable: false
+          description: synapse ID of master file view. 
+          example: syn23643253
+          required: true
+        - in: query
+          name: syn_manifest_file_name
+          schema:
+            type: string
+            nullable: false
+          description: Synapse storage manifest file name. 
```

Comment:
Change "Synapse storage manifest file name" to "Filename of dataset manifest"

Reply:
Thanks for the suggestion. I changed it to: Dataset manifest filename. For example, for Synapse this would be the manifest_filename in config.yml. 

Note: I used this parameter for /storage/project/datasets endpoint and /storage/dataset/files endpoint. The first endpoint is for getting all datasets in folder under a given storage project that the current user has access to, and the second one is for getting all the files in a given dataset.